flixel/FlxCamera.hx

```diff
@@ -493,6 +493,7 @@ class FlxCamera extends FlxBasic
 		itemToReturn.colored = colored;
 		itemToReturn.hasColorOffsets = hasColorOffsets;
 		itemToReturn.blending = blendInt;
+		itemToReturn.blend = blend;
```

Comment:
Don't we also need to check `blend` earlier in this function (lines 468-474) to check if two draw items are the same?

Reply:
Done.